R/comparePlot.R

```diff
@@ -335,55 +421,49 @@ comparePlot <- function(compareDF,
 #' }
 #'
 #' @importFrom assertthat assert_that
-#' @import magrittr
+#' @importFrom dplyr filter arrange bind_cols rename_with
+#' @importFrom magrittr set_rownames
 #'
 #' @export
 comparePrep <- function(contrastList,
```

Comment:
@agenius-mohammed-ali - Can you check with Connie if this needs to be a separate external function. As per my conversation with Connie in the meeting yesterday, she was saying we can avoid having prep functions and move  them to the main if it is small enough

Reply:
ok, though I think that does not apply on this function as it is not a helper method but an exported method